.github/workflows/ci.yml

```diff
@@ -0,0 +1,96 @@
+name: ci-workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: 'Repository info'
+        required: true
+      branch:
+        description: 'Name of the branch'
+        required: true
+      target_repository:
+        description: 'Repository name of the terratest'
+        required: true
```

Comment:
Should these all have defaults? i.e., The official fork repo and official Terratest repo?

Reply:
only target_repository can have a default value (which is gruntwork-io/terratest), I just pushed a commit to bring that default value